pages/index.tsx

```diff
@@ -67,6 +68,52 @@ export default function Home() {
           )}
           {userEmail}
         </div>
+
+        <h2 className="text-3xl">Our team</h2>
+        <div className="mt-8 flex flex-row flex-wrap justify-center gap-5 sm:gap-12 md:gap-24">
+          <div className="flex flex-col items-center">
+            <Image src={pavelImage} alt="personal" height={100} width={100} />
+            <p className="mt-4 mb-2 text-xl">Pavel</p>
+            <p>Taem Leader</p>
+          </div>
+          <div className="flex flex-col items-center">
+            <Image src={sadulloImage} alt="personal" height={100} width={100} />
+            <p className="mt-4 mb-2 text-xl">Sadullo</p>
+            <p>FE Developer</p>
+          </div>
+          <div className="flex flex-col items-center">
+            <Image src={tomasImage} alt="personal" height={100} width={100} />
+            <p className="mt-4 mb-2 text-xl">Tomas</p>
+            <p>FE Developer</p>
+          </div>
+        </div>
+
+        <div className="mt-6">
+          <h2 className="text-center text-3xl">React Course</h2>
+          <p className="text-center text-green-400 mb-6">Free | Online | in English</p>
+          <h3 className="text-xl mb-4">Target Audience</h3>
+          <p>
+            Students of the RS School from the 2022Q3, which has passed RS School Stage #2 as well
+            as new students with practical experience and knowledge of:
+          </p>
+          <ul className="list-outside list-disc ml-6">
+            <li>JavaScript</li>
+            <li>TypeScript</li>
+            <li>
+              Git, GitHub (clone, add, commit, push, pull, merge, rebase, working with Pull Request)
+            </li>
+            <li>NPM, Webpack</li>
+            <li>CSS3 / HTML5</li>
+            <li>Chrome DevTools, Figma</li>
+            <li>Understanding of the REST</li>
+          </ul>
+          <a
+            href="https://wearecommunity.io/events/rs-react-2023q1"
+            className="px-5 py-2 bg-pink-600 inline-block my-6 mx-auto hover:bg-pink-500 text-white"
+          >
+            Enroll
+          </a>
```

Comment:
Please make it look more like our other buttons/links with rounded corners and similar padding